assets/script.js

```diff
@@ -128,10 +129,43 @@ function displayRecipes(url) {
         let buttonEl = $("<button>")
         .addClass("btn btn-secondary modal-btns")
         // add attribute as an identifier for modal button to use to get recipie info from API
-        .attr("id", mealID)
+        .attr("id", recipeURL)
         .text("Cooking instructions");
 
-      recipeDiv.append(headerEl, recipeEl, buttonEl);
+      // Favourite button div
+      let favouriteDiv = $("<div>")
+      
+        // Favourite button created and added to card here
+        let buttonFavourite = $("<button>")
+        .addClass("btn btn-danger favourite-btns")
+        // add attribute as an identifier for favourite button
+        .attr("id", mealID)
+        .text("Favourite");
+        // Favourite icon created and added here
+        let iOne =$("<i>")
+        .addClass("glyphicon far fa-heart");
+        let iTwo =$("<i>")
+        .addClass("glyphicon fas fa-heart");
+
+      // Function- heart icon red if recipe has already been favourited and saved in local storage
+      function checkIcon (mealIDValue) {
+        mealIDValueText = (`${mealIDValue}`)
+      // get from local storage
+      var existingEntriesCheck = JSON.parse(localStorage.getItem("locations") || '[]');
```

Comment:
Nice work - I like the use of the OR operator.  We could rename "locations" to "favourites" to make it more explicit what we're storing. Also might be worth putting this in its own function to remove the code duplication on line 225

Reply:
Thanks! Yes you are right, I've changed that to favourites. I've also put line 154 code in a function as you suggested and used that function on line 154 and 225.

Reply:
Brill - thanks!